ADRs/0003-llm-benchmark-configuration.md

```diff
@@ -0,0 +1,25 @@
+# LLM-0003 LLM model configuration
```

Comment:
```suggestion
# LLM-0003 LLM model benchmark run configuration
```
As to me this is more about the config about how a benchmark is run on an LLM model instead of just how LLM models are run, right?

Reply:
right. resolved